db/schema.rb

```diff
@@ -292,6 +294,17 @@
     t.index ["charge_id"], name: "index_offence_dates_on_charge_id"
   end
 
+  create_table "other_charges", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
Could we have expanded on the existing charges table?

Reply:
You're right - I considered it when I started the ticket and decided to go this way because I didn't want to bloat the existing table with attributes that current charges don't have. I remember thinking that `charges` had a lot more attributes than it actually does for some reason... Somehow "other charges" felt distinct enough to warrant its own table. Looking at it now, I think reusing the existing table might be the better way to go, as long as I distinguish "current" charges from "other" charges.

Yeah, I feel way less convinced now about the approach I took originally 😅 I'll update to reuse `charges`.